tests/unit-tests/C/plugins/storage/common/README.rst

```diff
@@ -1,17 +1,21 @@
-Unit Test for common components of storage plugin
+*****************************************************
+Unit Test for Common Components of the Storage Plugin
+*****************************************************
 
 Require Google Unit Test framework
 
-Install with...
-* sudo apt-get install libgtest-dev
-* cd /usr/src/gtest
-* cmake CMakeLists.txt
-* sudo make
-* sudo cp \*.a /usr/lib
+Install with:
+::
+    sudo apt-get install libgtest-dev
+    cd /usr/src/gtest
+    cmake CMakeLists.txt
+    sudo make
+    sudo cp \*.a /usr/lib
```

Comment:
Why this cp command? There is a make install that uses the install program - much better for lots of reasons